scanapi/__main__.py

```diff
@@ -1,110 +1,3 @@
-import logging
+from .cli import main
```

Comment:
```suggestion
from scanapi..cli import main
```

Just a tiny suggestion to use an absolute import, as you did for the others, in order to follow PEP8 [recommendation](https://peps.python.org/pep-0008/#imports)

Reply:
Isn't there an extra dot here?

Reply:
Ohhhhh 🤣 🤦‍♀️ 

Sure, let me fix it. Great catch, thanks

Reply:
☺️

Reply:
sure! updated :)